### ISSUE TITLE
INSERT fails with non-null constraint violation

### ISSUE THREAD

**Issue by mpotra** [dbeaver/dbeaver#12786]
#### System information: 
- Operating system (distribution) and version: Fedora 31 - os.arch=amd64 os.name=Linux os.version=5.5.9-200.fc31.x86_64
- DBeaver version 21.1.0.202105300349
- org.jkiss.dbeaver.ce.feature (21.1.0.202105300349) "DBeaver Standalone"
- org.jkiss.dbeaver.runtime.feature (21.1.0.202105300349) "DBeaver Model"
- org.jkiss.dbeaver.standalone.feature (21.1.0.202105300349) "DBeaver Standalone"

#### Connection specification:
- Database name and version: PostgreSQL 13.2 (Debian 13.2-1.pgdg100+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 8.3.0-6) 8.3.0, 64-bit
- Driver name PostgreSQL JDBC Driver (org.postgresql:postgresql:RELEASE version 42.2.5)
- Do you use tunnels or proxies (SSH, SOCKS, etc)?  No
- org.jkiss.dbeaver.ext.postgresql (2.1.131.202105300349) "DBeaver PostgreSQL Support" [Active]
- org.jkiss.dbeaver.ext.postgresql.ui (1.0.34.202105300349) "DBeaver PostgreSQL Support UI" [Active]

#### Describe the problem you're observing:

Using the Editor UI Add Row feature to insert a row in the database, fails when not providing any column with a value.
All columns are defined to have default values, so the operation should insert a row even when no value is explicitly set by the user.

**Expected result:** DBeaver should use statement `INSERT INTO accouting.invoices DEFAULT VALUES;`
**Actual result:** DBeaver uses statement `INSERT INTO accounting.invoices (id, created_at, updated_at, deleted_at) VALUES (null, null, null, null)`

#### Steps to reproduce, if exist:
1. Create table with all columns having default values
2. In table Editor UI, use `Add Row` button, to insert a row (Alt+Insert)
3. Do not set/modify any of the column values of the row to be inserted.
4. Click on Save button (Ctrl+S)
5. Error is generated by PostgreSQL Database

Working alternative, but this is counter intuitive:
1. Create table with all columns having default values
2. In table Editor UI, use `Add Row` button, to insert a row (Alt+Insert)
3. Set the value for at least one column in the row to be inserted.
4. Click on Save button (Ctrl+S)
5. SQL Insert is ok

#### Include any warning/errors/backtraces from the logs

Table DDL:
```sql
CREATE TABLE accounting.invoices (
	id uuid NOT NULL DEFAULT uuid_generate_v4(),
	created_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
	updated_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
	deleted_at timestamp NULL,
	CONSTRAINT invoices_pkey PRIMARY KEY (id)
);
```

Error on inserting row with all defaults:
```
[92] ERROR:  null value in column "id" of relation "invoices" violates not-null constraint
[92] DETAIL:  Failing row contains (null, null, null, null).
[92] STATEMENT:  INSERT INTO accounting.invoices (id,created_at,updated_at,deleted_at)
VALUES ($1,$2,$3,$4)
```

**Comment by LonwoLonwo** [comment]
Looks like a good enhancement, thanks @mpotra!

small tip for the future me: "CREATE EXTENSION IF NOT EXISTS "uuid-ossp""

**Comment by LonwoLonwo** [comment]
![2l7LrcrXdy](https://user-images.githubusercontent.com/45152336/127448167-78a9de70-4bb1-4164-ab18-0aacb5516629.gif)

Implemented in DBeaver 21.1.4

Now we are using "INSERT INTO tableName DEFAULT VALUES" for an empty row with all default value cells.

**Comment by uslss** [comment]
verified. Related issue is created.
